spec/services/idv/phone_step_spec.rb

```diff
@@ -37,11 +37,16 @@
     Proofing::Mock::AddressMockClient::PROOFER_TIMEOUT_PHONE_NUMBER
   end
   let(:trace_id) { SecureRandom.uuid }
+  let(:attempts_tracker) do
+    stub_attempts_tracker
+    @irs_attempts_api_tracker
+  end
```

Comment:
I'd consider moving this into a `before` because `let` blocks are lazily loaded so the stubbing may happen in an unexpected order. Or use `let!` to force it to early evaluate

```suggestion
  before { stub_attempts_tracker }
  let(:attempts_tracker) { @irs_attempts_api_tracker }
```

Reply:
Done!